test/README.md

```diff
@@ -14,18 +14,24 @@ Our integration tests are based on functions from the samples project. To run in
    ```
    npm install -g azurite
    ```
-3. A local SQL Server instance - This is used by tests to verify that data is correctly added/fetched from the database when a test Function is run. You just need the server to be up and running, the tests will create the database and tables which will be cleaned up afterwards.
+3. A local SQL Server instance or an Azure SQL Database - This is used by tests to verify that data is correctly added/fetched from the database when a test Function is run. You just need the server to be up and running, the tests will create the database and tables which will be cleaned up afterwards.
```

Comment:
I'd just update this to be `A SQL Server instance used by tests to insert/read data by the test functions.` It doesn't really matter what type of servers they are and that makes it easier to add new supported servers later on.